tests/jballerina-unit-test/src/test/resources/test-src/types/stream/stream-value.bal

```diff
@@ -387,25 +387,29 @@ function testEmptyStreamConstructs() returns boolean {
     boolean testPassed = true;
     stream<int> emptyStream1 = new;
     stream<int> emptyStream2 = new stream<int>();
-    stream<int, never> emptyStream3 = new;
+    stream<int, ()> emptyStream3 = new;
     stream<int, error> emptyStream4 = new;
-    stream<int, never> emptyStream5 = new stream<int, never>();
+    stream<int, ()> emptyStream5 = new stream<int, ()>();
     stream<int, error> emptyStream6 = new stream<int, error>();
     var emptyStream7 = new stream<int>();
-    var emptyStream8 = new stream<int, never>();
+    var emptyStream8 = new stream<int, ()>();
     var emptyStream9 = new stream<int, error>();
 
     testPassed = testPassed && (emptyStream1.next() === ());
     testPassed = testPassed && (emptyStream2.next() === ());
     testPassed = testPassed && (emptyStream3.next() === ());
-    testPassed = testPassed && (emptyStream4.next() === ());
+
     testPassed = testPassed && (emptyStream5.next() === ());
-    testPassed = testPassed && (emptyStream6.next() === ());
+
     testPassed = testPassed && (emptyStream7.next() === ());
     testPassed = testPassed && (emptyStream8.next() === ());
-    testPassed = testPassed && (emptyStream9.next() === ());
 
-    // test the assignability of stream<int> and stream<int, never>
+    //todo will revisit during unbounded stream implementation
```

Comment:
Shall we add an issue to re-visit these?

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/29638